Dockerfile

```diff
@@ -1,15 +1,35 @@
 ARG POSTGRES_VERSION=12.1
 FROM postgres:${POSTGRES_VERSION}
-MAINTAINER rowe.andrew.d@gmail.com
+LABEL org.opencontainers.image.authors="rowe.andrew.d@gmail.com"
 
-RUN apt-get update && \
-    apt-get install -y cron && \
-    apt-get clean && rm -rf /var/lib/apt/lists/*
+EXPOSE 5432
+ENV PGUSER=postgres
+ENV PGDATA="/data"
 
-ADD *.sh /
-RUN chmod +x /*.sh
+COPY --chown=${PGUSER}:${PGUSER} \
+   [ "dump.sh", \
+     "entrypoint.sh", \
+     "/" ]
 
-VOLUME /dump
+USER root
+
+VOLUME [ "/dump", "/data" ]
+
+RUN \
+apt-get update && \
```

Comment:
I prefer these to be separate RUN items as it makes Docker cache layering easier